src/components/search/SearchResults.svelte

```diff
@@ -195,56 +198,56 @@ li {
 }
 
 .focused {
-    background-color: var(--bg-gray-01);
+  background-color: var(--bg-gray-01);
 }
 
 </style>
 
 <svelte:window bind:innerWidth={windowWidth} on:keydown={handleKeypress} />
 
 <Portal>
-{#if $store.searchIsActive && $store.searchQuery.length}
-  <div
-  id="telemetry-search-results"
-  style="left: calc({x}px + var(--space-base)); width: calc({width}px - var(--space-base));"
-    transition:fly={{ duration: 20, y: -10 }}
-    class="telemetry-results">
-      <div class="header-container">
-          {#if $searchResults.total}
-          <div class="header header--loaded" in:fly={{ x: -5, duration: 200 }}>
-              <div>found {$searchResults.results.length} of
-                  {formatTotal($searchResults.total)} probes
```

Comment:
This removal raises the following error on line 25. (GitHub won't allow me to comment on line 25.)

```
  25:5  error  'formatTotal' is assigned a value but never used  no-unused-vars
```